apps/src/code-studio/pd/form_components/FormController.jsx

```diff
@@ -258,9 +272,22 @@ export default class FormController extends React.Component {
       );
     }
 
+    const pageButtons = (
+      <Pagination
+        style={{
+          verticalAlign: 'middle',
+          margin: '0 10px'
+        }}
```

Comment:
style: our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react) prefers defining these styles in an object outside the component.